packages/headless/src/features/insight-user-actions/insight-user-actions-slice.test.ts

```diff
@@ -0,0 +1,201 @@
+import {
+  InsightUserActionsResponse,
+  UserActionType,
+} from '../../api/service/insight/user-actions/user-actions-response';
+import {
+  fetchUserActions,
+  registerUserActions,
+} from './insight-user-actions-actions';
+import {insightUserActionsReducer} from './insight-user-actions-slice';
+import {getInsightUserActionsInitialState} from './insight-user-actions-state';
+
+describe('insight user actions slice', () => {
+  const requestId = 'some-request-id';
+  const errorResponse = {
+    message: 'something bad happened',
+    statusCode: 400,
+    type: 'badluck',
+  };
+  const session = {
+    start: '2023-01-04T20:05:13.741Z',
+    end: '2023-01-04T21:00:42.741Z',
+    actions: [
+      {
+        actionType: UserActionType.CLICK,
+        timestamp: '2022-12-04T20:05:13.741Z',
+        eventData: {},
+        cause: 'resultClick',
+        searchHub: 'someSearchHub',
+        document: {
+          title: 'How to do that thing I need to do',
+          clickUri: 'https://www.insightfulwebsite.com/how-to',
+        },
+      },
+    ],
+  };
+
+  it('should have an initial state', () => {
+    expect(insightUserActionsReducer(undefined, {type: 'foo'})).toEqual(
+      getInsightUserActionsInitialState()
+    );
+  });
+
+  describe('#fetcUserActions', () => {
```

Comment:
```suggestion
  describe('#fetchUserActions', () => {
```

Reply:
your ability to spot these is unmatched! sorry for the typos 😅

Reply:
the original function actually had that typo but the search&replace doesn't go through 'strings' unfortunately.